Aptoide/aptoide-client-v8
# [ASV-1785] Removed ab testing from search;

**What does this PR do?**

This PR removes ab test from search;

**Database changed?**

 No

**How should this be manually tested?**

Make sure search still works as intended and no code related to this was left behind;

**What are the relevant tickets?**

  Tickets related to this pull-request: [ASV-1785](https://aptoide.atlassian.net/browse/ASV-1785)

**Code Review Checklist**
